src/NuGet.Core/NuGet.DependencyResolver.Core/ResolverUtility.cs

```diff
@@ -501,5 +504,16 @@ private static RemoteMatch CreateUnresolvedMatch(LibraryRange libraryRange)
                 Provider = null
             };
         }
+
+        private static void LogSourcesIfPackageNamespacesAreEnabled(string packageName, RemoteWalkContext context, IList<IRemoteDependencyProvider> remoteDependencyProviders)
```

Comment:
This method logs the sources returned by package namespaces configuration at `Debug` verbosity so that customers can trace which sources are queried while downloading a package.